js/script.js

```diff
@@ -356,6 +357,17 @@ function inputValidation() {
 const error = document.querySelectorAll('.error');
 
 contactForm.addEventListener('submit', (e) => {
+  /* Start local storage process */
+  // setup my user list and store it in local storage
+  const users = {
+    name: inputName.value,
+    email: inputEmail.value,
+    comment: inputComment.value,
+  };
+  data.push(users);
+  localStorage.setItem('users', JSON.stringify(data));
+  /* End local storage process */
+
```

Comment:
- [ ] Good job on storing your data in the localStorage 👏  👏  . But one of the requirements says when the user loads or reloads the page, the code checks if there's any saved data in the local storage and if so the input fields are pre-filled with the data. follow this example below.

```
window.onload = () => {
  let savedFormData = localStorage.getItem('contactFormInput');
  savedFormData = JSON.parse(savedFormData);
  // Check if the form data object is found on localStorage
  if (savedFormData) {
  // populate inputs values if data was found
  // ex: nameInput.value = savedFormData.name
  }
};